### DISCUSSION TITLE
[Idea] Allow no formating or Not when copy and paste

### DISCUSSION THREAD

**Discussion by Touxten** [PrestaShop/PrestaShop#36150] [Ideas]
### The problem

When you copy and paste  the text  in the description of a product is formatted, but it's not always useful.

### The solution

Allow formatting depend the context

### Additional context

A long french discussion :

https://friends-of-presta.slack.com/archives/C011CEW7Y6B/p1715688467557139

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by PrestaEdit** [comment]
https://github.com/PrestaShop/PrestaShop/pull/36151
